lib/frontend/app.js

```diff
@@ -126,12 +118,28 @@ module.exports = (appTokenGenerator) => {
       }
       Sentry.captureException(err)
     })
-    return res.status(errorCodes[err.message] || 400)
-      .render('github-error.hbs', {
-        info,
-        title: `${err.message} - GitHub + Jira integration`
-      })
-  })
+  }
+
+  const catchErrors = async (err, req, res, next) => {
+    if (process.env.NODE_ENV === 'development') {
+      return next()
```

Comment:
Same here. If we pass `err` into the context, the browser will show the error.

```suggestion
      return next(err)
```